genpack.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# generate `.tar.gz` packages from git repos
+
+# set PACKAGEURL
+set_pack_url() {
+	case "$1" in
+	tunasync)	PACKAGEURL=https://github.com/sjtug/tunasync.git ;;
```

Comment:
@htfy96 
 - Installing git in container consumes image space (~30MB), and is unnecessary.
 - Use external `.tar.gz` files make it possible to use older/modified/temporary versions of tunasync/mirror-web more easily

What do you mean by "simplifying dependency"?